tails_files/install.sh

```diff
@@ -162,13 +162,20 @@ if [ -f $XSESSION_RC ]; then
     fi
 fi
 
+# remove previous NetworkManager hook if present. it runs later than we'd like
+OLD_NM_HOOK=$TAILSCFG/custom-nm-hooks/99-tor-reload.sh
+if [ -f $OLD_NM_HOOK ]; then
+    rm -f $OLD_NM_HOOK > /dev/null 2>&1
+    rm -f $INSTALL_DIR/99-tor-reload.sh > /dev/null 2>&1
+fi
```

Comment:
Good point. I still prefer the conditional. Makes the intent more clear - this is supposed to be a one-time thing, cleaning up from an old version, and only needs to happen if that file is present. "Better explicit than implicit".